data/sql/updates/pending_db_world/rev_1630089599912305764.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1630089599912305764');
+
+-- Decreases the respawntime for Garrick Padfoot to 90 s
+UPDATE `creature` SET `spawntimesecs` = 90 WHERE `id` = 103 AND `guid` IN (80247);
```

Comment:
This is minor, but you don't need to use `IN` here since you're only updating one item. Using ``` `guid` = 80247``` will work just as well

Reply:
Yes, I know.